public/apps/configuration/constants.tsx

```diff
@@ -53,6 +53,24 @@ export const CLUSTER_PERMISSIONS: string[] = [
   'cluster:admin/tasks/cancel',
   'cluster:admin/tasks/test',
   'cluster:admin/tasks/testunblock',
+  'cluster:admin/opendistro/alerting/alerts/ack',
+  'cluster:admin/opendistro/alerting/alerts/get',
+  'cluster:admin/opendistro/alerting/destination/delete',
+  'cluster:admin/opendistro/alerting/destination/email_account/delete',
+  'cluster:admin/opendistro/alerting/destination/email_account/get',
+  'cluster:admin/opendistro/alerting/destination/email_account/search',
+  'cluster:admin/opendistro/alerting/destination/email_account/write',
+  'cluster:admin/opendistro/alerting/destination/email_group/delete',
+  'cluster:admin/opendistro/alerting/destination/email_group/get',
+  'cluster:admin/opendistro/alerting/destination/email_group/search',
+  'cluster:admin/opendistro/alerting/destination/email_group/write',
+  'cluster:admin/opendistro/alerting/destination/get',
+  'cluster:admin/opendistro/alerting/destination/write',
+  'cluster:admin/opendistro/alerting/monitor/delete',
+  'cluster:admin/opendistro/alerting/monitor/execute',
+  'cluster:admin/opendistro/alerting/monitor/get',
+  'cluster:admin/opendistro/alerting/monitor/search',
+  'cluster:admin/opendistro/alerting/monitor/write',
```

Comment:
Since this part starts with `cluster:admin/o`, I would suggest insert them between `cluster:admin/i` and `cluster:admin/r` to match the alphabetical order

Reply:
@yuxitang-amzn alphabetically ordered. thanks.